### PR TITLE
Fe feat/error page

### PR DESCRIPTION
**Pull request by jmartin777**
# Description

1.Created Error Page component (handles 404 / 500 / and other errors we can customize a message for.)
2. Installed most recent version of React Route
2.Updated App component for Home Page and Error Page routes (proper import paths)

## Type of change
-added functionality to the Error Page and Routed to the App component

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [ ] Refactoring

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/ErrorPage/ErrorPage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  if (error) {
+    if (error.status === 404) {
+      errorMessage = 'Page not found.';
+    } else if (error.status === 500) {
+      errorMessage = 'Internal server error.';
+    } else if (error.message) {
+      errorMessage = error.message;
+    }
+  }
```
Comment (andreasorensen) [inline src/Components/ErrorPage/ErrorPage.js:14]: I'm wondering if we can make the error message a little friendlier for the 5XX errors & also potentially handle ALL 5xx errors, something like: 

if (error.status >= 500 && < 600) ... that way ANY server errors are handled?

File: src/Components/App/App.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/error" element={<ErrorPage />} />
```
Comment (andreasorensen) [inline src/Components/App/App.js]: We are planning to conditionally render this error page, correct? At the top of the App return?

**Review by andreasorensen — APPROVED** [review]
Awesome job with the Error component.

File: src/Components/App/App.js (hunk 2 above)
Comment (jmartin777) [reply src/Components/App/App.js]: @andreasorensen  yes we are good catch!

**Review by Elise-Jones — APPROVED** [review]
This edit looks great.

**Pull request merged**